apache2.yaml

```diff
@@ -90,6 +90,12 @@ pipeline:
 
   - uses: strip
 
+  - uses: fetch
+    with:
+      uri: https://raw.githubusercontent.com/docker-library/httpd/master/2.4/alpine/httpd-foreground
+      expected-sha256: 003f4ca165cf0daa9721c907259d3ef6a7fd9017453079780c1ae91de96fa777
```

Comment:
for 2.4 you could use var-transform to compute 2.4.

Reply:
Done, thanks